site/data/docs/migrating-to-02.mdx

```diff
@@ -1,12 +1,72 @@
 ---
-title: Migrating to 0.2
+title: Migrating to >=0.2
 ---
 
-# Migrating to 0.2
+# Migrating to >=0.2
 
 ## Migrating RainbowKit
 
-RainbowKit has updated the `wagmi` peer dependency to `^0.4` which comes with a few [breaking changes](https://wagmi.sh/docs/migrating-to-03#04x-breaking-changes).
+### 0.3.x Breaking changes
+
+Removed the `chainId` parameter from `createConnector` on the `Wallet` type (Custom Wallets).
```

Comment:
How about something like this, to make it clear that most consumers don't need to care:

```suggestion
Removed the `chainId` parameter from `createConnector` on the `Wallet` type.

**Note that all built-in wallets are using the new API. Most consumers will be unaffected. This change only affects consumers that have created/consumed custom wallets.**
```